### DISCUSSION TITLE
Account not found.

### DISCUSSION THREAD

**Discussion by BrunBworks** [vercel/community#1736] [Help] [answered]
### Summary

Account was working this morning, now it cannot be found..
https://www.vercel.com/brunner-works

### Additional information

Uses GitHub integration with this account.

### Example

https://www.vercel.com/brunner-works

**Comment by jacky-lam** [comment]
I have the same problem too.

I get this error message when i try to login to my Vercel account via Github

<img width="897" alt="Screenshot 2023-03-09 at 17 16 07" src="https://user-images.githubusercontent.com/17092881/224104792-e149f9f8-e467-48cb-86ac-2fbe5f017264.png">

**Comment by amyegan** [comment, marked as answer]
Hi @BrunBworks. If you're having trouble logging in with GitHub, can you try logging in via email and let me know if that works for you? 

If that doesn't work, please send an email to [registration@vercel.com](mailto:registration@vercel.com) so we can keep your personal information private while we work with you to solve the problem. Be sure to include the email address and GitHub username that should be linked to your account to speed up the process 🙏

**Reply by envidi** [reply]
i cant login vercel with my account git hub . it alway display account not found

**Comment by alvinhmg** [comment]
I have also encountered the same problem.

**Reply by ThomasLG52** [reply]
I have also encountered the same problem.

**Comment by constmoon** [comment]
Is this an issue with not signing up? I linked my GitHub account through sign up and got logged in.

**Reply by charick05** [reply]
Thanks, you helped me

**Comment by 0xkuze** [comment]
I have the same error, I had to create a new account linked to my github, but all my projects disappeared.

**Comment by coderwyd** [comment]
I'm having the same problem, logging in through my github account keeps prompting me that my account can't be found!

**Comment by g0fcn** [comment]
你好，我也是显示Account not found，github账号g0fcn

**Reply by sj7510** [reply]
是直接登录的吗，先用github账号注册

**Comment by globalizatdev** [comment]
I have the same problem creating vercel account whith github using Neubox email to create all accounts.
